src/components/HelmetMetaData/index.tsx

```diff
@@ -0,0 +1,37 @@
+import React from "react";
+import { Helmet } from "react-helmet";
+import { useLocation } from "react-router-dom";
+import { IReactHelmet } from "../../interfaces/IReactHelmet";
+
+export default function HelmetMetaData(props: IReactHelmet) {
+  const location = useLocation();
+  const currentUrl = window.location.hostname + location.pathname;
+  const title = props.title
+    ? props.title
+    : "Daoseeder: World's First Decentralized crowdfunding platform";
```

Comment:
Change to the same text mentioned above.

Reply:
Removed this as we are now copying the url